script.js

```diff
@@ -101,45 +96,53 @@ function multiplication(array){
   wrong1 = getRandomInt(three-10, three+10);
   wrong2 = getRandomInt(wrong1-10, wrong1+10);
   console.log(one, "*", two, '=', three, "not", wrong1, "or", wrong2);
-  array.push ((one), (two), (three), (wrong1), (wrong2));
+  array.push ((three), (wrong1), (wrong2), (one), (two),);
   return(array)
-  
-//in an array, if you don't use quotation marks, that signifies a variable
-//since the following are not variables (they're not defined anywhere), it was giving an error
-//fixed that
-// Sorry Murat, we copied that array from the spec sheet and forgot to delete it after creating the animal sets. Thanks for cleaning it though! - Mas
-  
-  
-function multiplication(){
-  let one = getRandomInt(2, 16);
-  let two = getRandomInt(2, 16);
-  let three = one * two
-  console.log(one, "*", two, '=', three)
-  return(one, "*", two, '=', three)
-
 }
-// need to add the display functions on the page to display 1,2,3
+
 function subtraction (one, two, three){
   one = getRandomInt(10, 100);
   two = getRandomInt(1, one);
   three = one - two
   console.log(one, "-", two, '=', three)
   return(one, "-", two, '=', three)
 }
-// same here
  function addition (){
   let one = getRandomInt(1, 100);
   let two = getRandomInt(1, 100);
   let three = one + two
   return(one, "+", two, '=', three);
 }
   
+function shuffle(array) {
+  var currentIndex = array.length,  randomIndex;
```

Comment:
This is a variable with two values? Strange.
```suggestion
  let currentIndex = [array.length,  randomIndex];
```
Maybe it should be an array?